src/main/kotlin/no/nav/syfo/kafka/consumers/varselbus/domain/EsyfovarselHendelse.kt

```diff
@@ -82,6 +89,7 @@ enum class HendelseType {
     SM_ARBEIDSUFORHET_FORHANDSVARSEL,
     SM_VEDTAK_FRISKMELDING_TIL_ARBEIDSFORMIDLING,
     SM_FORHANDSVARSEL_MANGLENDE_MEDVIRKNING,
+    NL_DIALOGMOTE_INNKALLING_SVAR
```

Comment:
Kan vurdere å bare ha NL_DIALOGMOTE_SVAR, slik at det blir likt med de andre typene for dialogmøte